### DISCUSSION TITLE
bug(CodeEditor): 代码编辑器弹窗中显示异常问题

### DISCUSSION THREAD

**Discussion by HongFei-Ye** [dotnetcore/BootstrapBlazor#3188] [Q&A] [answered]
### Is there an existing issue for this?

- [X] I have searched the existing issues

### Describe the bug

CodeEditor 组件代码无法显示代码：CodeEditor 是放在一个组件里面，然后再页面里面使用弹窗弹出这个包含CodeEditor 的组件。
在我的项目里面首次点击弹窗能够显示出来，关闭弹窗后再次弹窗显示的时候就显示异常了，需要按下F12才能显示代码出来。

我做了一个复现示例：请查看zip文件与视频，这个复现示例里面，不知道为什么连首次弹窗代码区域也显示异常，需要按下F12才能显示代码出来。

### Expected Behavior

正常显示代码内容，不需要按下F12也能够显示出来。

### Steps To Reproduce

复现代码：
[BlazorApp1.zip](https://github.com/dotnetcore/BootstrapBlazor/files/14816051/BlazorApp1.zip)

复现视频：

https://github.com/dotnetcore/BootstrapBlazor/assets/131856535/813236e1-c97e-463a-b64f-1cc4c2763cc1

### .NET Version

8.0

**Comment by bb-auto** [comment]
@HongFei-Ye Thank you for reporting. We will give triage later.

**Comment by ArgoZhang** [comment, marked as answer]
@HongFei-Ye 

这个问题主要根源是由于组件所需样式未加载导致，由于组件没有高度，所以无法渲染，解决办法 F12 找到网络面板，禁止使用缓存后重新刷新页面即可解决。

组件更新版本 `8.0.2` 更新内部样式单词拼写错误

**Comment by kimdiego2098** [comment]
[BlazorApp1 (2).zip](https://github.com/user-attachments/files/17325729/BlazorApp1.2.zip)

实测问题还没解决

@ArgoZhang

**Reply by ArgoZhang** [reply]
@HongFei-Ye @kimdiego2098 更新到最新版本已修复
